Part3_InterruptRoutingSetupInAChipset.html

```diff
@@ -30,24 +30,24 @@ <h2>Theory</h2>
 
 	<img src="https://habrastorage.org/webt/mc/ra/hj/mcrahjn-owk9qrcmyiixcmhomcq.png" />
 
-	<p>On these pictures the mapping 'PCI device → PIR' is presented fairly abstract, actually it is a little bit more complicated. In a reality every PCI device has 4 interrupt lanes (INTA#, INTB#, INTC#, INTD#). Also every PCI device can have up to 8 so called 'functions' (which actually trigger interrupts) and each of these functions can connect to the one of the INTx# lane. Which INTx# interrupt will trigger each of the functions either is fixed in the hardware, or is defined by the device software configuration.<p/>
+	<p>On these pictures the mapping 'PCI device → PIR' is fairly abstract - in reality it is a little bit more complicated. In reality every PCI device has four interrupt lanes (INTA#, INTB#, INTC#, INTD#). Also, every PCI device can have up to 8 so called 'functions' (which actually trigger interrupts) and each of these functions can connect to one of the INTx# lanes. Which INTx# interrupt will trigger each of the functions is either fixed in the hardware, or is defined by the device software configuration.<p/>
 
 	<img src="https://habrastorage.org/webt/dk/ny/gh/dknyghwfo8gktn5l_lmmqb5awz0.png" />
 
-	<p>In a nutshell these functions are like separate logical blocks. For example one PCI device can have in itself 'Smbus controller' function, 'SATA controller' function and 'LPC bridge' function. From the point of OS every function is like a separate device with its own configuration space PCI Config.<p/>
+	<p>In a nutshell these functions are like separate logical blocks. For example, one PCI device can have in itself the 'SMBus controller' function, the 'SATA controller' function, and the 'LPC bridge' function. From the point of view of the OS every function is like a separate device with its own configuration space PCI Config.<p/>
 
-	<p>In a most simple (and a most common) case PCI device has only one function with its interrupt going to a lane INTA#. But in a common case device can have even more that 4 functions (as we've already said up to 8), and in this case it have to connect some of them to the same INTx# lane (PCI interrupts can share an interrupt lane). Also for PCI devices that are a part of a chipset, by writing to its special registers usually it is possible to declare which functions use which INTx# lanes (or don't use them at all).<p/>
+	<p>In the most simple (and most common) case, a PCI device has only one function with its interrupt going to the lane INTA#. In another common case the device can have even more that 4 functions (as we've already said up to 8), and in this case it has to connect some of them to the same INTx# lane (PCI interrupts can share an interrupt lane). Also for PCI devices that are part of a chipset, by writing to its special registers it is usually possible to declare which functions use which INTx# lanes (or don't use them at all).<p/>
```

Comment:
Can we use 'But generally' in a start of a second sentence? It is what I meant at least, it is not "an another common case"

Reply:
Ok, I think "In general" would work then (normally you try to avoid starting a sentence with "but").